dev/com.ibm.ws.security.wim.adapter.ldap_fat/publish/servers/com.ibm.ws.security.registry.ldap.fat.ids.extremeFailover/server.xml

```diff
@@ -27,7 +27,6 @@
 			<server host="${ldap.server.1.port}" port="379"/>
 		</failoverServers>
 		<failoverServers name="failoverLdapServersGroup2">
-			<server host="${ldap.server.4.name}" port="${ldap.server.4.port}"/>
```

Comment:
Should be: 

\<server host="${ldap.server.5.name}" port="${ldap.server.5.port}"/\>

Reply:
Hmm... perhaps it doesn't let the failover server be the same as the primary. I am not sure that this LDAP registry even gets used to connect as I don't believe the primary port is valid either.